opencensus-proto/Cargo.toml

```diff
@@ -1,10 +1,10 @@
 [package]
 name = "opencensus-proto"
-version = "0.1.0"
+version = { workspace = true }
 authors = ["The OpenCensus Authors"]
```

Comment:
another nice thing about this is that it highlights what crates are exceptions to the norm: this for example is vendored.